pkg/tsdb/mysql/mysql.go

```diff
@@ -32,6 +32,17 @@ func newMysqlQueryEndpoint(datasource *models.DataSource) (tsdb.TsdbQueryEndpoin
 		datasource.Url,
 		datasource.Database,
 	)
+
+	tlsConfig, err := datasource.GetTLSConfig()
+	if err != nil {
+		return nil, err
+	}
+
+	if tlsConfig.RootCAs != nil || len(tlsConfig.Certificates) > 0 {
+		mysql.RegisterTLSConfig(datasource.Name, tlsConfig)
```

Comment:
We should probably change this to datasource id so that it's unique per datasource.